lib/utils.go

```diff
@@ -1,1479 +1,99 @@
-// +build e2e_test
-
-// This is a file with e2e tests for C bindings written in library.go.
-// As a CGO file, it can't have `_test.go` suffix as it's not allowed by Go.
-// At the same time, we don't want this file to be included in the binaries.
-// This is why `e2e_test` tag was introduced. Without it, this file is excluded
-// from the build. Providing this tag will include this file into the build
-// and that's what is done while running e2e tests for `lib/` package.
-
 package main
 
-import "C"
 import (
-	"context"
+	"bytes"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"math/big"
-	"os"
-	"path/filepath"
-	"reflect"
-	"strconv"
 	"strings"
-	"testing"
 	"time"
-
-	gethcommon "github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/common/hexutil"
-	"github.com/ethereum/go-ethereum/core"
-	gethparams "github.com/ethereum/go-ethereum/params"
-	"github.com/stretchr/testify/require"
-
-	"github.com/status-im/status-go/geth/account"
-	"github.com/status-im/status-go/geth/common"
-	"github.com/status-im/status-go/geth/params"
-	"github.com/status-im/status-go/geth/signal"
-	"github.com/status-im/status-go/geth/transactions"
-	"github.com/status-im/status-go/geth/transactions/queue"
-	"github.com/status-im/status-go/static"
-	. "github.com/status-im/status-go/t/utils" //nolint: golint
 )
 
-const zeroHash = "0x0000000000000000000000000000000000000000000000000000000000000000"
-const initJS = `
-	var _status_catalog = {
-		foo: 'bar'
-	};`
-
-var testChainDir string
-var nodeConfigJSON string
-
-func init() {
-	testChainDir = filepath.Join(TestDataDir, TestNetworkNames[GetNetworkID()])
-
-	nodeConfigJSON = `{
-	"NetworkId": ` + strconv.Itoa(GetNetworkID()) + `,
-	"DataDir": "` + testChainDir + `",
-	"HTTPPort": ` + strconv.Itoa(TestConfig.Node.HTTPPort) + `,
-	"WSPort": ` + strconv.Itoa(TestConfig.Node.WSPort) + `,
-	"LogLevel": "INFO"
-}`
+// APIResponse generic response from API
```

Comment:
Would like to see this cleanup to also take some care on commenting. New `utils.go` is pretty small, so should be simple.

Reply:
Pushed some changes on formatting _(adding "." at the end of each function)_ and added function comments were missing :)